pkg/command/root.go

```diff
@@ -290,14 +331,59 @@ func newRootCmd() *cobra.Command {
 	return rootCmd
 }
 
+// determineVerbosityFromPluginCommandArgs processes the command line arguments for plugin commands.
+// It specifically looks for the '--verbose' flag and validates its value.
+// The function returns an error if the verbose flag is missing its value or the value is not an integer between 0 and 9.
+func determineVerbosityFromPluginCommandArgs(args []string) error {
```

Comment:
This logic is similar to what we are doing in `plugin_cmd.go`.
I wonder if we could consolidate it in `plugin_cmd.go` and call Cobra's `ParseFlags` on the `--verbose` flag args like helm does.

That way Cobra would fill the `verbose` variable instead of us having to do it here.
And we'll be prepared for any other future global flag we may add later on.